### DISCUSSION TITLE
Pnpm mono repro setup. 1 central admin?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [payloadcms/payload#3174] [Q&A]
If i where to take the pnpm mono repro where each site would be a package. Would it be possible to have 1 central admin to edit all my websites?

i would like to do something like admin.mydomain.ltd and be able to edit all my landing pages i want to build with payload.

Is this possible at the moment?
Any docs or code / repros i could look at for this?🙏

**Comment by denolfe** [comment]
It sounds like you're wanting to configure "multi-tenancy" aka one backend for multiple sites.

A common pattern is modeled out in this plugin: https://github.com/joas8211/payload-tenancy

This doesn't really have much to do with using pnpm or being in a monorepo besides the fact that you'd be able to share the exported TS types.
